include/cantera/thermo/ThermoPhase.h

```diff
@@ -1495,8 +1495,11 @@ class ThermoPhase : public Phase
     virtual void getParameters(int& n, doublereal* const c) const {
     }
 
-    //! Set equation of state parameters from an AnyMap phase description
-    void setParameters(const AnyMap& phaseNode);
+    //! Set equation of state parameters from an AnyMap phase description.
```

Comment:
Can this link to the `AnyMap` docs?

Reply:
I think Doxygen should do this automatically. See for example the docs for [`ThermoPhase::setState`](https://cantera.org/documentation/dev/doxygen/html/dc/d38/classCantera_1_1ThermoPhase.html#a310b4b2821895a9e11d8b98a949459c9) which contains automatically generated links to several classes:
https://github.com/Cantera/cantera/blob/8a591b4920e8ed525d41e94576f9241f5a890551/include/cantera/thermo/ThermoPhase.h#L1148-L1171
(and one spurious one to `namespace Cantera`.

Reply:
Great, I didn't know that Doxygen will autolink arbitrary words that it finds in the code. Guess you'd better not name a method with a common word :joy: